src/map/map.hpp

```diff
@@ -100,6 +101,7 @@ class display_data {
 
 	// map pixel -> province id
 	std::vector<uint16_t> province_id_map;
+	bool fow_enabled = false;
```

Comment:
why not make this a persistent setting accessible through the esc->graphics menu?